modules/fledgeForGpt.js

```diff
@@ -31,12 +32,10 @@ export function init(cfg) {
   }
 }
 
-export function addComponentAuctionHook(next, bidRequest, componentAuctionConfig) {
+export function addComponentAuctionHook(next, adUnitCode, componentAuctionConfig) {
```

Comment:
Would a hook not benefit more from seeing the original bidRequest ?

Reply:
possibly, but we don't always have it. In prebid server we may have multiple requests (the fledge config references an imp, but there can be any number of bidders (requests) for that imp). With `allowUnknownBidderCodes` we may also have zero.